abis/mlibc/in.h

```diff
@@ -53,6 +53,19 @@ struct ip_mreq {
 	struct in_addr imr_interface;
 };
 
+// These structs are copied from linux and modified to work, they may be severely wrong
```

Comment:
Why did they need to be "modified to work"?

Reply:
Do these structs match linux now/are they not modified to work anymore?

Reply:
yes, these do match linux now